tests/php/View/AttributesHTMLTest.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace SilverStripe\View\Tests;
+
+use SilverStripe\Dev\SapphireTest;
+
+class AttributesHTMLTest extends SapphireTest
+{
+
+    public function singleAttributeProvider(): array
```

Comment:
```suggestion
    public function provideGetAttribute(): array
```
Follow the naming conventions for data providers